rioxarray/_io.py

```diff
@@ -880,7 +880,7 @@ def open_rasterio(
         coord_name = "band"
         coords[coord_name] = np.asarray(riods.indexes)
 
-    has_gcps = hasattr(riods, "gcps") and riods.gcps != ([], None)
+    has_gcps = hasattr(riods, "gcps") and riods.gcps[0] != []
```

Comment:
Thoughts about this?
```suggestion
    has_gcps = hasattr(riods, "gcps") and riods.gcps[0]
```

Reply:
Fine by me!